src/current/v23.2/physical-cluster-replication-monitoring.md

```diff
@@ -14,14 +14,15 @@ docs_area: manage
 - [`SHOW VIRTUAL CLUSTER ... WITH REPLICATION STATUS`](#sql-shell) in the SQL shell.
 - The [Physical Replication dashboard](#db-console) on the DB Console.
 - [Prometheus and Alertmanager](#prometheus) to track and alert on replication metrics.
+- [`SHOW EXPERIMENTAL_FINGERPRINTS`](#data-verification) to verify data up to a point in time is correct on the standby cluster.
```

Comment:
Here, and in other sections below, I think I would go with "at a point in time" rather than "up to a point in time" since the current command doesn't look at history and only the single point in time.